<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PixivNotifier</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="pnlLogin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>470</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>470</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lePassword">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>320</y>
       <width>160</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
font: 9pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="placeholderText">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lePixivID">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>280</y>
       <width>160</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
font: 9pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="placeholderText">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLogin">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>360</y>
       <width>160</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
background-color: rgb(170, 170, 255);
font: 9pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <zorder>lePassword</zorder>
    <zorder>lePixivID</zorder>
    <zorder>btnLogin</zorder>
    <zorder>pnlMain</zorder>
    <zorder>pnlMain</zorder>
    <zorder>pnlMain</zorder>
   </widget>
   <widget class="QFrame" name="pnlMain">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>470</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>470</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>112</y>
       <width>300</width>
       <height>340</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane{
	border:none;
}
QTabWidget::tab-bar{
        alignment:left;
}
QTabBar::tab{
	background:transparent;
	color:white;
	min-width:25ex;
	min-height:40px;
}
QTabBar::tab:hover{
	background:rgb(255, 255, 255, 100);
}
QTabBar::tab:selected{
	border-color: white;
	background:white;
	color:green;
}</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tbMain">
      <attribute name="title">
       <string/>
      </attribute>
     </widget>
     <widget class="QWidget" name="tbLoved">
      <attribute name="title">
       <string/>
      </attribute>
     </widget>
     <widget class="QWidget" name="tbCreator">
      <attribute name="title">
       <string/>
      </attribute>
      <widget class="QScrollBar" name="vsclBar">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>0</y>
         <width>16</width>
         <height>300</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tbMessage">
      <attribute name="title">
       <string/>
      </attribute>
     </widget>
    </widget>
    <widget class="QLabel" name="userAvatar">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>8</y>
       <width>96</width>
       <height>96</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 48px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
